drivers/SCsub

```diff
@@ -6,6 +6,9 @@ env.drivers_sources=[]
 #env.add_source_files(env.drivers_sources,"*.cpp")
 env_drivers.Append(CPPPATH=["vorbis"])
 
+if (env["theoralib"]=="yes" and (env["x86_opt_gcc"] or env["x86_opt_vc"])):
+	env_drivers.Append(CCFLAGS=["-DOC_X86_ASM"]);
```

Comment:
I don't think that's the proper fix. An `Export('env')` at the end of `drivers/theora/SCsub` would have done the trick.